lib/client.js

```diff
@@ -1397,7 +1397,10 @@ function write (client, request) {
 
     socket.cork()
     socket.write(`${header}content-length: ${contentLength}\r\n\r\n`, 'ascii')
-    socket.write(body)
+    const endCallback = request.onBodyChunkSent
+      ? request.onBodyChunkSent.bind(request, contentLength, contentLength)
```

Comment:
Is `bind` really necessary in this case?

Reply:
I could create an arrow function instead, if it makes a difference.